app/routes/send-endringsmelding.module.css

```diff
@@ -0,0 +1,19 @@
+.fyllSide {
+  height: 30rem;
+  padding: 0.5rem;
+  display: grid;
+  align-items: center;
+}
+
+.innholdKonteiner {
+  max-width: 37.5rem;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  gap: 5rem;
+  align-items: center;
+
+  @media (min-width: 768px) {
+    width: 37.5rem;
+  }
+}
```

Comment:
Er det mulig å gjøre dette felles for sidene på noen måte?

Reply:
Et alternativ er å lage et komponent som wrapper rundt all inholdet på nettsiden og inneholder stylingen. Veit ikkje om den må leggest rundt innholdet på kvar side eller om remix har en ekvivalent til `layouts` 🧐

Reply:
Se commit 22431ac for et nytt komponent for å sentrere alt.
Har ikkje lagt til opp til at index siden gjør det. Da ser koden slik ut når du wrapper den: 
```javascript
export default function SendEndringsmelding() {
  const tekster = useTekster(ESanitySteg.SEND_ENDRINGER);

  return (
    <InnholdKonteiner>
      <TekstBlokk
        tekstblokk={tekster.overskrift}
        typografi={TypografiTyper.StegHeadingH1}
      />
    </InnholdKonteiner>
  );
}

```